packages/camera/camera/example/ios/RunnerTests/CameraTestUtils.h

```diff
@@ -0,0 +1,22 @@
+// Copyright 2013 The Flutter Authors. All rights reserved.
+// Use of this source code is governed by a BSD-style license that can be
+// found in the LICENSE file.
+
+@import camera;
+
+NS_ASSUME_NONNULL_BEGIN
+
+@interface CameraTestUtils : NSObject
```

Comment:
Please don't make fake classes that aren't actually classes just to hold what are effectively just functions; ObjC isn't Java, and has support for free-standing functions.

Reply:
I changed to functions but kept the filename `CameraTestUtils`. There doesn't seem to be a rule about the file name for collection of functions in our [style guide](https://google.github.io/styleguide/objcguide.html#file-names). What do you think?